catalog/plugin/catalog-plugin-metacardbackup/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/MetacardBackupPlugin.java

```diff
@@ -96,28 +88,28 @@
             return processRequest;
         }
 
-        if (StringUtils.isEmpty(outputDirectory)) {
+        if (CollectionUtils.isEmpty(storageBackupPlugins)) {
             throw new PluginExecutionException(
-                    "Unable to delete backup ingested metacard; no output directory specified.");
+                    "Unable to delete backup ingested metacard; no metacard backup storage provider configured.");
         }
 
         List<ProcessDeleteItem> processUpdateItems = processRequest.getProcessItems();
         for (ProcessDeleteItem processUpdateItem : processUpdateItems) {
             Metacard metacard = processUpdateItem.getMetacard();
-            deleteBackupIfPresent(metacard.getId());
+            try {
+                for (MetacardBackupStorageProvider storageProvider : storageBackupPlugins) {
+                    storageProvider.deleteData(metacard.getId());
+                }
+            } catch (IOException | MetacardBackupException e) {
+                LOGGER.debug("Unable to delete backed up metacard data for metacard: {}",
```

Comment:
Good catch. Fixed this one and that storage side to try each provider independently.